Documentation/common-issues.md

```diff
@@ -458,6 +459,37 @@ rbd
 ```
 Now when a host is restarted, the module should be loaded automatically.
 
+# Rook Agent rbd module missing error
+## Symptoms
+* Rook Agent in `Error` or `CrashLoopBackOff` status when deploying the Rook operator with `kubectl create -f operator.yaml`:
+```
+$kubectl -n rook-ceph-system get pod
+NAME                                 READY     STATUS    RESTARTS   AGE
+rook-ceph-agent-gfrm5                0/1       Error     0          14s
+rook-ceph-operator-5f4866946-vmtff   1/1       Running   0          23s
+rook-discover-qhx6c                  1/1       Running   0          14s
+```
+* Rook Agent logs contain below messages:
+```
+2018-08-10 09:09:09.461798 I | exec: Running command: cat /lib/modules/4.15.2/modules.builtin
+2018-08-10 09:09:09.473858 I | exec: Running command: modinfo -F parm rbd
+2018-08-10 09:09:09.477215 N | ceph-volumeattacher: failed rbd single_major check, assuming it's unsupported: failed to check for rbd module single_major param: Failed to complete 'check kmod param': exit status 1. modinfo: ERROR: Module rbd not found.
+2018-08-10 09:09:09.477239 I | exec: Running command: modprobe rbd
+2018-08-10 09:09:09.480353 I | modprobe rbd: modprobe: FATAL: Module rbd not found.
+2018-08-10 09:09:09.480452 N | ceph-volumeattacher: failed to load kernel module rbd: failed to load kernel module rbd: Failed to complete 'modprobe rbd': exit status 1.
+failed to run rook ceph agent. failed to create volume manager: failed to load kernel module rbd: Failed to complete 'modprobe rbd': exit status 1.
+```
+
+## Solution
+From the log message of Agent, we can see that the `rbd` kernel module is not available in the current system, neither as a builtin nor a loadable external kernel module.
+
+In this case, you have to [re-configure and build](https://www.linuxjournal.com/article/6568) a new kernel to address this issue, there're two options:
```

Comment:
are we very confident that this scenario can't be solved by running `modprobe rbd` on the host machine in some circumstances? will a kernel rebuild always be required?

Reply:
@jbw976 For a running system without the `rbd` module (either builtin or loadable), the `modprobe rbd` on the host machine will definitely fail, the only way to solve this is to rebuild/upgrade the kernel, thus the volume manager can work normally. see the comments of issue #1985